### PR TITLE
Disable de-federation (to avoid running into a scalability issue).

### PR DESCRIPTION
**Pull request by fisx**
https://wearezeta.atlassian.net/browse/WPB-4668

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: changelog.d/1-api-changes/WPB-4668-disable-defederation [hunk 1]
```diff
@@ -0,0 +1 @@
+Disable de-federation (to avoid running into a scalability issue).
```
Comment (akshaymankar) [inline changelog.d/1-api-changes/WPB-4668-disable-defederation:1]: I think we should also delete other changelog entries which talk about defederation

**Review by akshaymankar — COMMENTED** [review]
Why not delete the endpoints?

File: changelog.d/1-api-changes/WPB-4668-disable-defederation (hunk 1 above)
Comment (fisx) [reply changelog.d/1-api-changes/WPB-4668-disable-defederation:1]: i would have left that as an exercise for the release curator, but fair, will do!

**Comment by fisx** [comment]
> Why not delete the endpoints?

it's only one end-point.  we still want to be able to add remotes to the table, or update the search policy, or list them.

**Comment by fisx** [comment]
but yes, i'll do that.

File: changelog.d/1-api-changes/WPB-4668-disable-defederation [hunk 2]
```diff
@@ -0,0 +1 @@
+Remove de-federation (to avoid a scalability issue).
```
Comment (akshaymankar) [inline changelog.d/1-api-changes/WPB-4668-disable-defederation:1]: Seems noisy for people reading the changelog to read about what hasn't changed.

**Review by akshaymankar — APPROVED** [review]

File: changelog.d/1-api-changes/WPB-4668-disable-defederation (hunk 2 above)
Comment (fisx) [reply changelog.d/1-api-changes/WPB-4668-disable-defederation:1]: defederation has been introduced in 4.36.0.

**Comment by fisx** [comment]
federator i-tests fail to compile now:

```
test/integration/Test/Federator/IngressSpec.hs:66:24: error:
    Ambiguous occurrence ‘originDomain’
    It could refer to
       either the field ‘originDomain’ of record ‘IntegrationConfig’,
              imported from ‘Test.Federator.Util’ at test/integration/Test/Federator/IngressSpec.hs:42:1-26
           or the field ‘originDomain’ of record ‘BackendTwoConfig’,
              imported from ‘Test.Federator.Util’ at test/integration/Test/Federator/IngressSpec.hs:42:1-26
   |
66 |               Domain . originDomain . view teTstOpts $ env
   |                        ^^^^^^^^^^^^

test/integration/Test/Federator/IngressSpec.hs:152:21: error:
    Ambiguous occurrence ‘originDomain’
    It could refer to
       either the field ‘originDomain’ of record ‘IntegrationConfig’,
              imported from ‘Test.Federator.Util’ at test/integration/Test/Federator/IngressSpec.hs:42:1-26
           or the field ‘originDomain’ of record ‘BackendTwoConfig’,
              imported from ‘Test.Federator.Util’ at test/integration/Test/Federator/IngressSpec.hs:42:1-26
    |
152 |     originDomain <- originDomain . view teTstOpts <$> input
 ```

**Comment by fisx** [comment]
`/integration` is in utter disarray, but maybe the rest is done?

**Comment by fisx** [comment]
```
4 failed tests: 
 - Brig.testCrudFederationRemotes
 - Conversation.testAddingUserNonFullyConnectedFederation
 - Conversation.testDynamicBackendsNotFullyConnected
 - Federation.testNotificationsForOfflineBackends
```

**Comment by fisx** [comment]
```
federator-integration: AesonException "Error in $: When parsing the record IntegrationConfig of type Test.Federator.Util.IntegrationConfig the key backendTwo was not present."

CallStack (from HasCallStack):

  error, called at test/integration/Test/Federator/Util.hs:145:100 in main:Test.Federator.Util
```

i think we missed a spot under `/charts/` somewhere?

**Pull request merged**